crm_task/models/crm_lead.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Adgensee - Vincent Garcies
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class LeadTaskCount(models.Model):
+    _inherit = 'crm.lead'
+
+    taskcount = fields.Integer(
+        string="Task", compute='_compute_task_count',
+        required=False)
+
+    @api.depends('partner_id')

```

Comment:
Why does it depend on the partner_id if the customer_id isn't used in the '_compute_task_count' function?
Could the taskconunt field  be calculated like this?
```
def _compute_task_count(self):
    self.taskcount = len(self. task_ids)
```

Reply:
I just made the changes you suggest